joint_limits:
  arm_1_joint:
    has_velocity_limits: true
    max_velocity: 0.2      # rad/s
    has_acceleration_limits: true
    max_acceleration: 0.3  # rad/s^2
  arm_2_joint:
    has_velocity_limits: true
    max_velocity: 0.2
    has_acceleration_limits: true
    max_acceleration: 0.3
  arm_3_joint:
    has_velocity_limits: true
    max_velocity: 0.2
    has_acceleration_limits: true
    max_acceleration: 0.3
  arm_4_joint:
    has_velocity_limits: true
    max_velocity: 0.2
    has_acceleration_limits: true
    max_acceleration: 0.3
  arm_5_joint:
    has_velocity_limits: true
    max_velocity: 0.2
    has_acceleration_limits: true
    max_acceleration: 0.3
  arm_6_joint:
    has_velocity_limits: true
    max_velocity: 0.2
    has_acceleration_limits: true
    max_acceleration: 0.3
  arm_7_joint:
    has_velocity_limits: true
    max_velocity: 0.2
    has_acceleration_limits: true
    max_acceleration: 0.3